custom_components/tuya_local/devices/woods_cortina_airconditioner.yaml

```diff
@@ -0,0 +1,95 @@
+name: Woods AC Cortina Silent 12k
+products:
+  - id: bfc0e42617f4fdfef7t6ra
+    name: Wood's Cortina Silent 12k
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - value: "off"
+          dps_val: false
+          icon: "mdi:hvac-off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - value: cool
+              dps_val: "0"
+              icon: "mdi:snowflake"
+            - value: dry
+              dps_val: "2"
+              icon: "mdi:water"
+            - value: fan_only
+              dps_val: "3"
+              icon: "mdi:fan"
+            - value: heat_cool
```

Comment:
Yeah I agree, it is called ECO mode on the device itself, which doesn't map to any of the build-in HVAC modes. However, according to the [HA dev doc](https://developers.home-assistant.io/docs/core/entity/climate/) there is an Eco preset that can be used. Do you know how I can reference the Eco preset? If I just write  "eco" I think it doesn't work, because it doesn't use the translation key.

Reply:
No, you can't mix preset_modes with hvac_modes in HA.  I guess heat_cool is OK then, but maybe add a comment saying it is really eco.